### ISSUE TITLE
laravel 5.5/5.6 updates

### ISSUE THREAD

**Issue by illuminate3** [jeremykenedy/laravel-material-design#13]
Hi, I was wondering if you have plans to update this or create a new project based on 5.5/5.6?

**Comment by jeremykenedy** [comment]
Sadly, due to time constraints, I do not have it in my plans to update this project to the most recent codebase. 

Although I will accept pull requests from anyone who wants to help out :)

**Comment by illuminate3** [comment]
I was using laravel-boiler plate but have been wanting something more modular (built on modules), as laravel as much as possible and came across your logger package. Honestly, I haven't installed this yet but have been digging through your repos the last 2 days.

I'm going to tinker around and see what I can do for pull requests.

What would be your thoughts on adding vue to the mix? 

Life - full of time constraints. ;-)

**Comment by jeremykenedy** [comment]
I love Vue and am all for it!

**Comment by illuminate3** [comment]
I am a vue noob, so do you have any recommended projects that could be used to combine?

**Comment by jeremykenedy** [comment]
Yeah,

The current users page Is using a vue component on laravel-auth:

https://github.com/jeremykenedy/laravel-auth/blob/master/resources/assets/js/app.js#L21

https://github.com/jeremykenedy/laravel-auth/blob/master/resources/assets/js/bootstrap.js

https://github.com/jeremykenedy/laravel-auth/blob/master/resources/assets/js/components/UsersCount.vue

https://github.com/jeremykenedy/laravel-auth/blob/master/resources/views/pages/admin/active-users.blade.php

https://github.com/jeremykenedy/laravel-auth/blob/master/.env.example#L43 - note: since then Laravel 5.6 has made never version have this pull from the .env in bootstrap.js

It's a very basic example of a component (this one uses pusher.js).

**Comment by jeremykenedy** [comment]
The hope was that one day I could break this and laravel auth up into composer packages and have it assemble the project from the packages created but that's a big hope :)

Thats how I ended up putting together some of the packages I have was by pulling them from the laravel-auth project then converting them into packages.

This repo was a lesson in the crazy naming conventions of MDL and all the fun vanilla js:)

**Comment by illuminate3** [comment]
Note: I'm not the best programmer but I do know what I want and what good code looks like.

I had a chat with the caffeinated modules guy about packages vs modules. Many early laravel adopters bought whole scale into packages. Like the first big wave of everybody chanting "repository method" but now we have Jeffery saying "do what works and makes sense".

I like how you broke most things up.

I really want to have the ability to keep packages as "libraries" that can work standalone with default views and also have modules that would pull several packages together to form functionality. A basic idea here would be a "Users Module" which provides contacts, roles, and information which is really several packages that are being extended and with their views over written with your own templates.

The modules could be downloaded on install from the admin.

For me Plug-ins are additional functionality rather than a full package or module. Widgets are even smaller and more specialized.

TLDR; Just wanted to give you an idea on how I view packages, modules, plugins and widgets. I also want that click and play ability.

Vue: The reason why I am thinking of moving towards view is because I thought it would be a great way to clean up templates and off load certain views to API. I'm old school PHP (is a tempalting language).

Thanks for all those links! I'm gonna install this later tonight after work.

**Comment by jeremykenedy** [comment]
I have updated the codebase to Laravel 5.6. :)
